### ISSUE TITLE
Fast Roots aiming tries to avoid allies and yourself

### ISSUE THREAD

**Issue by projectNoob** [crawl/crawl#3232]
**PLATFORM and OS or BROWSER:**

webtiles, kelbi, chrome
**DCSS VERSION AND GAME INFORMATION:**

tested in 0.30 and 0.31(trunk)
**DESCRIPTION:**

I evoked a wand of roots and tried to kill gnolls in front of me. As I know Fast Roots does not friendly fire, but the auto-aiming mechanics still tried to avoid hitting me by aiming the space left-behind the gnoll.
**EXTRA FILES:**

**Comment by PleasingFungus** [comment]
Yeah, it's very annoying. I tried to fix this when I first added wand of roots but had no success; hopefully someone else has more luck than I do!

**Comment by sosloow** [comment]
Apparently @SentientSupper fixed this in https://github.com/SentientSupper/crawl/commit/6ff3a70e3eb300b298bdfadea7cdb2cf4279f54d

I cherry-picked the commit, and it seems to work as intended. Maybe this bug fix can be merged separately from the rest of the PR, while it's still being considered for merge?

Aiming a wand of roots:
![Screenshot from 2024-07-28 07-14-35](https://github.com/user-attachments/assets/c23c617e-f180-44da-9e52-0a167071b2a2)

Aiming a spell, that will damage the player:
![Screenshot from 2024-07-28 07-14-20](https://github.com/user-attachments/assets/0653b58c-0bbb-432e-b695-b145d6cadf8c)

**Comment by DracoOmega** [comment]
This has been fixed since various targeting overhauls in 0.33
